indico/modules/rb/models/locations.py

```diff
@@ -60,6 +60,9 @@ class Location(db.Model):
             ondelete='SET NULL'
         )
     )
+    map_url_template = db.Column(
+        db.String
```

Comment:
`nullable=False, default=''` - it's nice not having to deal with None values, especially when adding a wtform for this in the future